conformant = "Unsupported"
notes = """
Does not understand @dataclass_transform.
"""
output = """
File "dataclasses_transform_class.py", line 23, in <module>: Arguments to dataclass_transform not supported yet [not-supported-yet]
File "dataclasses_transform_class.py", line 57, in <module>: Invalid keyword argument other_name to function Customer1.__init__ [wrong-keyword-args]
         Expected: (self, id, name, name2)
  Actually passed: (self, id, name, other_name)
File "dataclasses_transform_class.py", line 82, in <module>: Arguments to dataclass_transform not supported yet [not-supported-yet]
File "dataclasses_transform_class.py", line 103, in <module>: Invalid keyword argument id to function GenericCustomer.__init__ [wrong-keyword-args]
         Expected: (self)
  Actually passed: (self, id)
File "dataclasses_transform_class.py", line 106, in <module>: Arguments to dataclass_transform not supported yet [not-supported-yet]
"""
